### PR TITLE
Added edit profile and view profile pages

### PR DESCRIPTION
**Pull request by osdagoso**
Relevant to issue #15 .

DETAILED CHANGE LOG:
- Added Edit Profile page
- Added field validation to page (kudos to @katiearriagam for basic guidelines on this one)
- Added View Profile page (w/ working Edit Profile)
- Added basic styling to pages
- Added success and failure messages for edits

TO-DO LIST:
- Once authentification is added, apply to view profile (loading first user's data at the moment)

### CODE REVIEW

File: src/Template/Users/view.ctp [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+	$this->assign('title', $user->fname . " " . $user->lname);
+	$this->set('bodyClass', 'bg-white view');
+?>
+
+<?php
+	$this->Form->setTemplates([
```
Comment (hecerinc) [inline src/Template/Users/view.ctp]: You're not using forms in this view. Why are you setting templates for the forms?

File: src/Template/Users/view.ctp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+			{{content}} <small id={{errorId}} class="editError hiddenElements">{{errorMessage}}</small></div>'
+	]);
+?>
+
+<div class="container view-container">
+	<?= $this->Flash->render() ?>
+	<h1 id="name-title">
+	<?php
+		echo $this->Text->insert(':first :last',['first' => $user->fname, 'last' => $user->lname]);
+		echo $this->Html->image('edit-pencil-black.png', [
```
Comment (hecerinc) [inline src/Template/Users/view.ctp]: If you can do this in svg, all the better [nice-to-have] [low-priority]

Comment (hecerinc) [reply src/Template/Users/view.ctp]: Also you might want to add a tooltip or label to guide users

File: src/Template/Users/view.ctp [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+				"alt" => "Edit",
+				'url' => ['action' => 'edit', $user->id]]);
+		//<div>Icons made by <a href="http://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
+	?>
+	</h1>
+	<div class = "row"><div class = "col">
+	<?= $this->Html->div('view-info pink','Correo electr&oacute;nico') ?>
+	</div><div class = "col">
```
Comment (hecerinc) [inline src/Template/Users/view.ctp]: Correct indentation

File: webroot/css/style.css [hunk 4]
```diff
@@ -378,6 +380,48 @@ header.main-nav {
...
+}
+
+.editError {
+	color: #de268f;
+	display: block;
+	float: left;
+	margin-top: 10px;
+}
+
+.hiddenElements {
```
Comment (hecerinc) [inline webroot/css/style.css]: Use the `.hidden` class already in the stylesheet

File: webroot/css/style.css [hunk 5]
```diff
@@ -538,3 +581,49 @@ ul#menu li {
...
+	width: 20px;
+	height: 20px;
+}
+
+div.message {
+    text-align: center;
+    cursor: pointer;
+    display: block;
+    font-weight: normal;
+	font-size: 17px;
```
Comment (hecerinc) [inline webroot/css/style.css]: Fix indentation (all tabs)

File: webroot/js/editUser.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
+var isNotValid = false;
```
Comment (hecerinc) [inline webroot/js/editUser.js]: Avoid global variables. Make the method return instead.

**Comment by hecerinc** [comment]
@osdagoso checa los merge conflicts para poder mergearlo please :)

**Comment by osdagoso** [comment]
@hecerinc Ya quedaron 👍 creo que por mis últimos commits (nothing major, really) se requiere que lo apruebes otra vez pero ya no debe de haber merge conflicts.

**Pull request merged**